tests/utils/test_task_group.py

```diff
@@ -1263,3 +1263,41 @@ def test_mapped_task_group_id_prefix_task_id():
 
     dag.get_task("t1") == t1
     dag.get_task("g.t2") == t2
+
+
+def test_get_task_dict():
+    with DAG("test_dag", start_date=pendulum.parse("20200101")) as dag:
+        with TaskGroup("section_1") as tg1:
+            EmptyOperator(task_id='task1')
+
+        with TaskGroup("section_2") as tg2:
+            task2 = EmptyOperator(task_id="task2")
+            task3 = EmptyOperator(task_id="task3")
+            mapped_bash_operator = BashOperator.partial(task_id="get_tag_template_result").expand(
```

Comment:
One super minor nit: This task_id feels a little specific/out of place compared to the others (looks like it might have come from the `example_datacatalog_tag_templates.py` system test?). Maybe just `task4` or `bash_task`? But this isn't a blocking suggestion, approving otherwise :smile:

Reply:
Yes good idea thanks, I'm going to update that :)

Reply:
Done, `bash_task` looks nice, ty :)